docs/USAGE.md

```diff
@@ -69,6 +69,76 @@ Ensures your code changes never ship without proper test coverage. The agent rev
 
 See [examples](../examples/custom_workflows) of more custom workflow configurations.
 
+## 🔧 Slash Commands
+
+Slash commands allow you to predefine specific behaviors for the agent. When a user includes a slash command in their instruction, the agent will use the corresponding prompt to guide its response.
+
+### Using Slash Commands
+
+Simply include the slash command in your comment along with your request:
+
+```text
+@h2ogpte /review this code and check for security issues
+```
+
+```text
+@h2ogpte /explain how this authentication flow works
+```
+
+```text
+@h2ogpte /plan a feature to add user authentication
+```
+
+You can use multiple slash commands in a single instruction:
+
+```text
+@h2ogpte /review and /explain this code
+```
+
+### Default Slash Commands
+
+The action comes with three default slash commands:
+
+- **`/explain`** - Provides clear and accurate code explanations
+- **`/review`** - Performs comprehensive code reviews
+- **`/plan`** - Creates implementation plans without writing code
+
+### Custom Slash Commands
+
+You can define your own slash commands in your workflow configuration. Each command requires:
+
+- **`name`**: The command identifier (must start with `/`)
+- **`prompt`**: The system prompt that guides the agent's behavior when this command is used
+
+**Example workflow configuration:**
```

Comment:
`**Example workflow configuration:**` should be a 4th level heading so it's properly rendered in the GitHub table of contents.